src/pages/nft/claim/index.vue

```diff
@@ -777,6 +777,7 @@ export default {
     }
   },
   methods: {
+    ...mapActions(['fetchCollectedNFTClassesByAddress']),
```

Comment:
I think NFT mixin already provided this action

https://github.com/likecoin/liker-land/blob/08ef670f2c6a8bd6bcbe62256309f9d2a50c944f/src/mixins/nft.js#L763

Reply:
[🔥 Remove duplicated action](https://github.com/likecoin/liker-land/pull/1631/commits/eebffbf34ef7603e3a4a090d419883fc69d77146)

Reply:
[🔥 Remove duplicated action](https://github.com/likecoin/liker-land/pull/1631/commits/eebffbf34ef7603e3a4a090d419883fc69d77146)